tests/integration/components/light-table-test.js

```diff
@@ -189,3 +189,29 @@ test('passed in components can have computed properties', function(assert) {
 
   assert.equal(this.$('.custom-row.is-active').length, 0, 'none of the items are active');
 });
+
+test('onScroll', function(assert) {
+  let done = assert.async();
+  let table = new Table(Columns, createUsers(10));
+
+  this.setProperties({
+    table,
+    currentScrollY: 0,
+    onScroll() {
+      assert.ok(true, 'onScroll worked');
```

Comment:
The actual value was slightly different, probably because of row heights, table size, etc. Just wanted to check the event handler was properly called on scroll.

Should I improve the test to also check for position or is this test good enough?